### ISSUE TITLE
Why is Failed Login Count in Member Editable?

### ISSUE THREAD

**Issue by sunnysideup** [silverstripe/silverstripe-framework#8334]
## Affected Version
SS4.1
## Description
Why is Failed Login Count in Member Editable?

![image](https://user-images.githubusercontent.com/167154/44332439-2bc7f180-a4c0-11e8-9322-2c98f16bb5d0.png)

Maybe make the field readonly or hide it altogether or on a different tab with other security info.

**Comment by dhensby** [comment]
Hi, thanks for taking the time to post to our issue tracker.

The issue tracker is reserved for reporting bugs and we don't provide support via the issue tracker. There are great [community support options](http://www.silverstripe.org/community/#introduction) where you can get help including [our forum](https://forum.silverstripe.org), [Stack Overflow (remember to tag as silverstripe)](http://stackoverflow.com/questions/tagged/silverstripe) and even [IRC](http://www.silverstripe.org/community/contributing-to-silverstripe/irc-channel/) or [Slack](http://silverstripe.org/slack).

Sorry we can't be of more help and good luck resolving your problem!

**Comment by dhensby** [comment]
I don't think there's a consistent reason this is editable; but I *think* making it editable would allow an admin to reinstate access to an account with too many failed logins.. maybe.

**Comment by sunnysideup** [comment]
Hi @dhensby Daniel,

Thank you for your swift reply.  I think this is actually a _**UX bug**_ of sorts, the "Failed Login Count" is out of place where is sits now.  It should go on a separate tab with other security related information such as Last Login Date, IP Address or whatever else we want and can show.  I dont think I get that across very well, do you understand what I mean? 

Imagine being a website owner and you are setting up three new users (your colleagues), you enter their first name, last name, email password and then you are asked to enter the Failed Login Count... that just comes across really weird and unpolished, like the developers forgot to move it to its rightful place.

**Comment by dhensby** [comment]
> Imagine being a website owner and you are setting up three new users (your colleagues), you enter their first name, last name, email, password, etc... and then you are asked to enter the Failed Login Count...

true - pretty rubbish.

**Comment by robbieaverill** [comment]
I think we could probably make it readonly. We probably need @clarkepaul's input here

**Comment by sunnysideup** [comment]
Apart from readonly (I think that makes sense), it should go to a different position. It is totally in the wrong place.

**Comment by clarkepaul** [comment]
Yeah its only editable so an admin can reset the count for others to log in again. 

How about the field is readonly but we add a reset button so that the count cannot be accidentally increased and block people out (or block yourself out by increasing your own count, and try to log back in again). 

I agree it doesn't make sense in the same order as the rest of the fields on the profile page but I wouldn't move it to a tab unless there are other fields. We could do something fancier to combine it with the password area but I just don't think its valuable enough to do something too custom here.

**Comment by sunnysideup** [comment]
The way I see it is that we set up a tab with statsy security info:

Member Since: __________
Last Updated: ____________
Last Login: ___
etc....

**Comment by robbieaverill** [comment]
Noting that out of the box the "Last login" data is not monitored

**Comment by sunnysideup** [comment]
Yeah - sorry, that was just an example of what could be included

What about:

- Created.Nice
- LastEdited.Nice
- PasswordExpiry.Nice
- LockedOutUntil.Nice
- FailedLoginCount
All read only?

What about `Locale`?

**Comment by clarkepaul** [comment]
I'd support those. Is "LastEdited" when the profile was last updated, or when that member last edited something in the CMS? 
If the former, we might need to communicate that through the label a bit clearer (eg. Member updated).

**Comment by maxime-rainville** [comment]
@clarkepaul `LastEdited` is a standard field all DataObject gets. It gets updated whenever a change gets written to a DataObject. So the answer would be "when the profile was last updated".

Although, we usually don't show that value.

**Comment by lerni** [comment]
I like clarkepauls idea having it readonly with a reset button for admin - this solves the issue, but dislike adding another tab. Showing more is doable with an extension but for me less is more - for tabs & fields.

**Comment by robbieaverill** [comment]
Yep +1 from me too
